deployments/prob140/config/common.yaml

```diff
@@ -0,0 +1,73 @@
+jupyterhub:
+  scheduling:
+    userScheduler:
+      enabled: true
+  proxy:
+    https:
+      letsencrypt:
+        contactEmail: rylo@berkeley.edu
+  hub:
+    extraConfig: |
```

Comment:
This can move to hub/values.yaml I think, since it's common to datahub & prob140?

Reply:
Done. Since this touches the datahub deployment can you double-check my hub/values.yaml?